Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM registry.centos.org/centos/centos:7
 
-ENV F8A_WORKER_VERSION=ca7d91f \
```

Comment:
I would recommend to add direct git pypi packages into requirements.in, something like https://github.com/kubesecurity/osa-api-server/blob/bee4ed3bb81dda7012fbe29c4f268202e373c481/requirements.in#L4. It will help you to organise all dependencies at one place instead of spreading it across everywhere

Reply:
will note it in TODO's